fw-child/resources/js/cdc.js

```diff
@@ -1078,29 +1078,6 @@
                     url: choro_path,
                     dataType: 'json',
                     success: function (data) {
```

Comment:
Is the `success` function still necessary since it's a kind of noop?

Reply:
Indeed... looks *promising*